src/Runner.Listener/JobDispatcher.cs

```
@@ -346,7 +346,38 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, string orc
                 }
 
                 var term = HostContext.GetService<ITerminal>();
-                term.WriteLine($"{DateTime.UtcNow:u}: Running job: {message.JobDisplayName}");
+
+                string workflowName = string.Empty;
+                if(message.Variables.TryGetValue("system.workflowFilePath", out var workflowPath))
+                {
+                    workflowName = workflowPath.Value.Split('/').LastOrDefault().Trim();
+                }
+
+                string repositoryName = string.Empty;
+                if(message.ContextData.TryGetValue("github", out var context))
+                {
+                    var githubContext = context as Pipelines.ContextData.DictionaryContextData;
+                    if(githubContext.TryGetValue("repository", out var repository))
```

We should fail silently if these casts fail, `githubContext?.TryGetValue(`

---

Ok

---

Done